core/src/main/java/pl/netigen/core/netigenapi/NetigenSplashFragment.kt

```diff
@@ -132,6 +131,10 @@ abstract class NetigenSplashFragment<ViewModel : NetigenViewModel> : NetigenFrag
     }
 
     private fun initGDPRFragment() {
+        if (!canCommitFragments) {
+            consentNotShowed = true
+            return
+        }
         val fragment = netigenMainActivity.supportFragmentManager.findFragmentByTag(GDPR_POP_UP_TAG) as GDPRDialogFragment?
         if (fragment != null) {
             gdprDialogFragment = fragment
```

Comment:
gdprDialogFragment should be treated as notNull here so that if there's an error we will be sure where it is. Though it's not quite possible to throw a NPE here